scripts/specializations/CpCourseManager.lua

```diff
@@ -257,57 +258,75 @@ function CpCourseManager:getOffsetFieldWorkCourse()
 end
 
 function CpCourseManager:getCourses()
-    local spec = self.spec_cpCourseManager 
+    local spec = self.spec_cpCourseManager
     return spec.courses
 end
 
 function CpCourseManager:hasCourse()
-    local spec = self.spec_cpCourseManager 
+    local spec = self.spec_cpCourseManager
     return next(spec.courses) ~= nil
 end
 
+function CpCourseManager:calculateCourseVisibility()
```

Comment:
@tn4799 Could you please add a ``cp`` to the function, for example like: ``cpCalculateCourseVisibility()``.
These functions are registered directly into the vehicle and every CP function should have a unique cp identifier in there for better mod compatibility and cleaner code :)